config/NaoH25V40.srdf

```diff
@@ -3,88 +3,112 @@
     This is a format for representing semantic information about the robot structure.
     A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
 -->
-<robot name="naoH25V40">
+<robot name="NaoH25V40">
     <!--GROUPS: Representation of a set of joints and links. This can be useful for specifying DOF to plan for, defining arms, end effectors, etc-->
     <!--LINKS: When a link is specified, the parent joint of that link (if it exists) is automatically included-->
     <!--JOINTS: When a joint is specified, the child link of that joint (which will always exist) is automatically included-->
     <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
-    <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names  -->
-
-    <group name="arm_left">
-	<chain base_link="torso" tip_link="LForeArm" />
+    <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
+    <group name="left_arm">
+        <chain base_link="torso" tip_link="l_wrist" />
     </group>
-    <group name="arm_right">
-	<chain base_link="torso" tip_link="RForeArm" />
+    <group name="right_arm">
+        <chain base_link="torso" tip_link="r_wrist" />
     </group>
-
     <group name="head">
-	<chain base_link="torso" tip_link="gaze" />
+        <chain base_link="torso" tip_link="Head" />
     </group>
     <group name="leg_left">
-	<chain base_link="torso" tip_link="l_sole" />
+        <chain base_link="torso" tip_link="l_sole" />
     </group>
     <group name="leg_right">
-	<chain base_link="torso" tip_link="r_sole" />
+        <chain base_link="torso" tip_link="r_sole" />
     </group>
-    <group name="hand_left">
-        <joint name="LWristYaw" />
+    <group name="left_hand">
```

Comment:
did you change the name of the group on purpose ? 
Can we keep the previous naming? which was: CHAINNAME_SIDE, e.g. leg_right, arm_right...